common/src/main/scala/com/gu/media/model/VideoAsset.scala

```diff
@@ -19,4 +19,5 @@ object VideoAsset {
   implicit val formatYouTube: Format[YouTubeAsset] = Jsonx.formatCaseClass[YouTubeAsset]
   implicit val formatSelfHosted: Format[SelfHostedAsset] = Jsonx.formatCaseClass[SelfHostedAsset]
   implicit val format: Format[VideoAsset] = Jsonx.formatSealed[VideoAsset]
+  val firstFrameImageSuffix = ".0000000.jpg"
```

Comment:
might want to move this into `VideoSource` when you've rebased from main (that's where the captions suffix is now stored)